src/identities/dto/register-identity.dto.ts

```diff
@@ -0,0 +1,44 @@
+/* istanbul ignore file */
+
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+import { IsDate, IsOptional, IsString, ValidateNested } from 'class-validator';
+
+import { PermissionedAccountDto } from '~/accounts/dto/permissioned-account.dto';
+import { TransactionBaseDto } from '~/common/dto/transaction-base-dto';
+
+export class RegisterIdentityDto extends TransactionBaseDto {
+  @ApiProperty({
+    description: 'Account address for which to create an Identity',
+    example: '5grwXxxXxxXxxXxxXxxXxxXxxXxxXxxXxxXxxXxxXxxXxxXx',
+  })
+  @IsString()
+  readonly targetAccount: string;
+
+  @ApiPropertyOptional({
+    description: 'Secondary Accounts and their permissions to be added to the Identity',
+    type: PermissionedAccountDto,
+    nullable: true,
+  })
+  @IsOptional()
+  @ValidateNested({ each: true })
+  @Type(() => PermissionedAccountDto)
+  secondaryAccounts?: PermissionedAccountDto[];
+
+  @ApiPropertyOptional({
+    description:
+      'Issue a CDD claim for the created DID, completing the onboarding process for the Account',
+    type: 'boolean',
+    example: false,
+  })
+  readonly createCdd?: boolean;
```

Comment:
I think this should be a required parameter here. Its better to be explicit, and we aren't worried about avoiding a breaking change like the SDK